libraries/MultiThreading/examples/Semaphore/Semaphore.ino

```diff
@@ -0,0 +1,52 @@
+// Please read file README.md in the folder containing this example.
```

Comment:
This example is not working on my ESP32 board.

`Backtrace: 0x40087e51:0x3ffb9150 0x40084557:0x3ffb9230 0x400d2343:0x3ffb9270 0x400d157d:0x3ffb9290 0x400d1903:0x3ffb92b0 0x400d1915:0x3ffb92d0 0x400d12b3:0x3ffb92f0`